recipe/meta.yaml

```diff
@@ -1,25 +1,25 @@
 {% set name = "pyee" %}
-{% set version = "9.1.0" %}
+{% set version = "10.0.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: a2287af31d6ce6e662ebf8034ea2c3e38dea02ebac63f6a41db23825bf6649d4
+  sha256: b767899d653f0f2a9a35ddb4fcce1e7555fd89758d9d9ae017329fa46f242d3d
 
 build:
   noarch: python
   number: 0
-  script: cd {{ RECIPE_DIR }} && {{ PYTHON }} builder.py
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
For belt-and-suspenders, I still recommend `--no-deps --no-build-isolation`, even though CI _should_ take care of this.